src/Kros.AspNetCore/Authorization/ApplicationBuilderExtensions.cs

```diff
@@ -43,5 +43,13 @@ public static class ApplicationBuilderExtensions
             this IApplicationBuilder app,
             Func<GatewayJwtAuthorizationOptions> configureOptions)
             => app.UseMiddleware<GatewayAuthorizationMiddleware>(configureOptions());
+
+        /// <summary>
+        /// Add JwtBearerClaims middleware.
+        /// </summary>
+        /// <param name="app"><see cref="IApplicationBuilder"/> where middleware is added.</param>
```

Comment:
Please add a info to comment, that `JwtSecurityTokenHandler` must be registered in DI.